pkg/frontend/transport/handler.go

```diff
@@ -226,6 +226,8 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 		"fetched_index_bytes", numIndexBytes,
 		"sharded_queries", stats.LoadShardedQueries(),
 		"split_queries", stats.LoadSplitQueries(),
+		"insight", "true",
+		"user_id", userID,
```

Comment:
Note that log already includes `user=userID`, can we use that instead of duplicating the value in two distinct keys?

Reply:
Good catch.